lib/utils.pm

```diff
@@ -1057,6 +1058,23 @@ sub is_boot_encrypted {
     return 1;
 }
 
+=head2 need_passphrase_again_in_grub
+
+ need_passphrase_again_in_grub();
+
+With newer grub2 (in TW and SLE15-SP6 currently), if root disk is encrypted,
+entering the passphrase in GRUB2 is enough. The key is passed on during boot,
+so it's not asked for a second time.
+We need to enter the passphrase again if there are separate partitions encrypted
+without LVM configuration (cr_swap,cr_home etc).
+
+=cut
+
+sub need_passphrase_again_in_grub {
+    return 1 if (is_leap || is_sle('<15-sp6') || is_leap_micro || is_sle_micro || is_alp || !get_var('LVM', '0'));
```

Comment:
Can be simplified:

`return is_leap || is_sle('<15-sp6') || is_leap_micro || is_sle_micro || is_alp || !get_var('LVM', '0');`